combination_test/run_test.sh

```
@@ -1,4 +1,4 @@
-#!/bin/bash
+█!/bin/bash
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)